gensim/test/test_fasttext.py

```diff
@@ -533,6 +533,18 @@ def test_sg_hs_training_fromfile(self):
                 overlap_count, 2,
                 "only %i overlap in expected %s & actual %s" % (overlap_count, expected_sims_words, sims_gensim_words))
 
+    def test_cbow_hs_training_fixedwindowsize(self):
```

Comment:
You can avoid this repetitiveness with parameterized tests (see my suggestion above).

Reply:
Nice! I did not know about this pytest feature. I will definitely deploy it to reduce the redundant code.